### DISCUSSION TITLE
State of WASM C API support?

### DISCUSSION THREAD

**Discussion by FGasper** [bytecodealliance/wasm-micro-runtime#662] [General]
Hello,

What is the state of WAMR’s support of the standard WASM C API?

I see sample code for it, but the main page lists it as “post-MVP”.

Thank you!

**Comment by lum1n0us** [comment]
Here is [a supported API list](https://github.com/bytecodealliance/wasm-micro-runtime/blob/main/doc/wasm_c_api.md) and we are working on *frames* now.  Plus, you will find [samples](https://github.com/bytecodealliance/wasm-micro-runtime/tree/main/samples/wasm-c-api/src) in the repo.
